### DISCUSSION TITLE
Startup Error for Elide 6.1.0

### DISCUSSION THREAD

**Discussion by white3km** [yahoo/elide#2515] [General]
Hello, I'm trying to migrate from 6.0.6 to 6.1.0 but I'm seeing an error on startup. Looks like the ElideSubscriptionScanningConfiguration is running even though graphql and subscriptions are disabled.

Error:
Parameter 1 of constructor in com.yahoo.elide.spring.config.ElideSubscriptionScanningConfiguration required a bean of type 'javax.jms.ConnectionFactory' that could not be found.

Steps to Reproduce:
- Checkout https://github.com/yahoo/elide-spring-boot-example
- Update pom.xml by removing spring-boot-starter-artemis and artemis-jms-server and set elide to 6.0.6
- Comment out Subscription in ArtifactGroup.java
- Update application.yml, toggle enabled to false for elide.graphql and elide.subscription then remove activemq settings
- Application runs, no errors
- Update pom.xml change elide to 6.1.0
- Application fails on startup, with error

**Comment by aklish** [comment]
Will take a look

**Reply by aklish** [reply]
Good catch.  Here is the fix:

https://github.com/yahoo/elide/pull/2521
